_non-cosmos/ethereum/assetlist.json

```diff
@@ -1382,6 +1382,237 @@
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/ena.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/ena.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/ena.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/ena.svg"
+      }
+    },
+    {
+      "description": "Ethena USDe is a synthetic dollar protocol built on Ethereum.",
+      "type_asset": "erc20",
+      "address": "0x4c9edd5852cd905f086c759e8383e09bff1e68b3",
+      "denom_units": [
+        {
+          "denom": "0x4c9edd5852cd905f086c759e8383e09bff1e68b3",
+          "exponent": 0,
+          "aliases": [
+            "usde-wei"
+          ]
+        },
+        {
+          "denom": "usde",
+          "exponent": 18
+        }
+      ],
+      "base": "0x4c9edd5852cd905f086c759e8383e09bff1e68b3",
+      "name": "Ethena USDe",
+      "display": "usde",
+      "symbol": "USDe",
+      "coingecko_id": "ethena-usde",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usde.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usde.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usde.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usde.svg"
+      }
+    },
+    {
+      "description": "A Liquid Restaking Token (LRT) and Strategy Manager for EigenLayer.",
+      "type_asset": "erc20",
+      "address": "0xbf5495efe5db9ce00f80364c8b423567e58d2110",
+      "denom_units": [
+        {
+          "denom": "0xbf5495efe5db9ce00f80364c8b423567e58d2110",
+          "exponent": 0,
+          "aliases": [
+            "ezeth-wei"
+          ]
+        },
+        {
+          "denom": "ezeth",
+          "exponent": 18
+        }
+      ],
+      "base": "0xbf5495efe5db9ce00f80364c8b423567e58d2110",
+      "name": "Renzo Restaked ETH",
+      "display": "ezeth",
+      "symbol": "ezETH",
+      "coingecko_id": "renzo-restaked-eth",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/ezeth.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/ezeth.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/ezeth.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/ezeth.svg"
```

Comment:
It it my believe that all the SVGs uploaded here were created by PNG to SVG conversion, all of which have artefacts and seem to be low quality. I think we should omit these, and allow the actual SVG to be uploaded later if they are found.